src/pages/[id]/status.tsx

```diff
@@ -82,32 +83,34 @@ const SaksStatusView: NextPage = () => {
 
             {!mustLogin && (
                 <>
-                    <DriftsmeldingAlertstripe />
-
-                    <ForelopigSvarAlertstripe />
-
-                    <SoknadsStatus />
-
-                    {erPaInnsyn && <Oppgaver />}
-
-                    {kommune != null && kommune.erInnsynDeaktivert && (
-                        <>
-                            <StyledPanel className="panel-luft-over">
-                                <Heading level="2" size="medium">
-                                    {t("vedlegg.tittel")}
-                                </Heading>
-                            </StyledPanel>
-                            <StyledPanel className="panel-glippe-over">
-                                <VedleggView fiksDigisosId={fiksDigisosId} />
-                            </StyledPanel>
-                        </>
-                    )}
-                    {(kommune == null || !kommune.erInnsynDeaktivert) && (
-                        <ArkfanePanel
-                            historikkChildren={<Historikk fiksDigisosId={fiksDigisosId} />}
-                            vedleggChildren={<VedleggView fiksDigisosId={fiksDigisosId} />}
-                        />
-                    )}
+                    <FilUploadSuccesfulProvider>
+                        <DriftsmeldingAlertstripe />
+
+                        <ForelopigSvarAlertstripe />
+
+                        <SoknadsStatus />
+
+                        {erPaInnsyn && <Oppgaver />}
+
+                        {kommune != null && kommune.erInnsynDeaktivert && (
+                            <>
+                                <StyledPanel className="panel-luft-over">
+                                    <Heading level="2" size="medium">
+                                        {t("vedlegg.tittel")}
+                                    </Heading>
+                                </StyledPanel>
+                                <StyledPanel className="panel-glippe-over">
+                                    <VedleggView fiksDigisosId={fiksDigisosId} />
+                                </StyledPanel>
+                            </>
+                        )}
+                        {(kommune == null || !kommune.erInnsynDeaktivert) && (
+                            <ArkfanePanel
+                                historikkChildren={<Historikk fiksDigisosId={fiksDigisosId} />}
+                                vedleggChildren={<VedleggView fiksDigisosId={fiksDigisosId} />}
+                            />
+                        )}
+                    </FilUploadSuccesfulProvider>
```

Comment:
Bruker alle disse komponenete som er wrappet i FilUploasSuccesfulProvider contexten? Det er lurt å bare wrappe de som er nødvendige for å ikke rendre formye unødvendig (jeg har ikke helt oversikt over hvor den ble brukt).

Reply:
Ok skjønner var usikker på hva som var best practice. Når det gjelder komponentene som bruker FilUploadSuccessfulProvider er det <oppgaver/> og  <VedleggView/>.

Reply:
I `vedleggView` så er det `EttersendelseView` som bruker det